logstash-tester.sh

```diff
@@ -50,47 +41,41 @@ error() {
 }
 
 run_docker() {
-    action=$1
-    configtest=$2
 
     if ! hash docker 2> /dev/null; then
         error "Can't find the Docker executable. Did you install it?"
     fi
-
-    rootdir=`dirname $0`
-
+    
     echo "====> Build docker image for test"
-    docker build -t gaspaio/logstash-tester \
-        --build-arg LST=$rootdir \
-        --build-arg FILTER_CONFIG=$3 \
-        --build-arg PATTERN_CONFIG=$4 \
-        --build-arg FILTER_TESTS=$5 \
-        --build-arg PATTERN_TESTS=$6 \
-        -f $rootdir/Dockerfile .
+    docker build -t gaspaio/logstash-tester $BUILD_ARGS -f `pwd`/Dockerfile .
```

Comment:
Haven't tested but `pwd`/Dockerfile wont work if the script executing logstash-tester.sh is in another dir than the Dockerfile, which is my use case. hence line 60 which you removed.

I'm not a bash expert, so I may be wrong on this.